frontend/review/src/components/GraphBox.js

```diff
@@ -0,0 +1,501 @@
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
+import { defaults, Scatter } from "react-chartjs-2";
+
+let addDropDate = [];
```

Comment:
why is this not const?

Reply:
I want to be able to access it outside of the functional component. After I get the data in the functional component, I want to change the value of it. Is there a better way of doing this?

Reply:
hmm I could just push it in